tests/portals/github/test_manager.py

```diff
@@ -0,0 +1,154 @@
+    manager = portals.GithubManager(control)
+    apikey = os.environ.get("GITHUB_ACCESS_TOKEN", None)
+    package = manager.read_package(user=user, repo=repo, apikey=apikey)
+    assert json.loads(package.to_json()) == {
+        "name": "test-repo-without-datapackage",
+        "resources": [
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/capitals.csv"
+            },
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/countries.csv"
+            },
+        ],
+    }
+
+
+@pytest.mark.vcr
+def test_github_parser_read_without_datapackage(options_without_packages):
+    user = options_without_packages.pop("user")
+    repo = options_without_packages.pop("repo")
+    control = portals.GithubControl()
+    manager = portals.GithubManager(control)
+    with pytest.raises(FrictionlessException) as excinfo:
+        manager.read_package(user=user, repo=repo)
+    error = excinfo.value.error
+    assert error.message == "Package not found"
+
+
+@pytest.mark.vcr
+def test_github_parser_read_only_csv(options_without_dp):
+    user = options_without_dp.pop("user")
+    repo = options_without_dp.pop("repo")
+    control = portals.GithubControl()
+    manager = portals.GithubManager(control)
+    apikey = os.environ.get("GITHUB_ACCESS_TOKEN", None)
+    package = manager.read_package(user=user, repo=repo, apikey=apikey)
+    assert json.loads(package.to_json()) == {
+        "name": "test-repo-without-datapackage",
+        "resources": [
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/capitals.csv"
+            },
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/countries.csv"
+            },
+        ],
+    }
+
+
+@pytest.mark.vcr
+def test_github_parser_read_other_file_types(options_without_dp):
+    user = options_without_dp.pop("user")
+    repo = options_without_dp.pop("repo")
+    control = portals.GithubControl(formats=["csv", "xlsx"])
+    manager = portals.GithubManager(control)
+    apikey = os.environ.get("GITHUB_ACCESS_TOKEN", None)
+    package = manager.read_package(user=user, repo=repo, apikey=apikey)
+    assert json.loads(package.to_json()) == {
+        "name": "test-repo-without-datapackage",
+        "resources": [
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/capitals.csv"
+            },
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/countries.csv"
+            },
+            {
+                "path": "https://raw.githubusercontent.com/fdtester/test-repo-without-datapackage/master/data/student.xlsx"
+            },
+        ],
+    }
+
+
+# Write
+
+
+@pytest.mark.vcr
```

Comment:
There should be a VCR option to exclude sensitive information from the cassete. See this commit - https://github.com/frictionlessdata/frictionless-py/blob/7db069431b2f5f60c518e5c05dfedafa53504f28/data/cassettes/test_github_parser_write_package_file.yaml - our Github Token is a part of the cassete. Nothing to worry about as Github automatically revoked this token when it hit the public repo but we need a way to exclude it from the cassete